sys/net/crosslayer/ng_netreg/ng_netreg.c

```diff
@@ -16,9 +16,8 @@
 #include <string.h>
 
 #include "clist.h"
-#include "net/ng_netreg.h"
-#include "net/ng_nettype.h"
 #include "utlist.h"
+#include "net/ng_netbase.h"
```

Comment:
The depending modules should not include this header IMHO.
Every module should also at least include their header, otherwise this looks weird.